joblib/testing.py

```diff
@@ -43,13 +43,12 @@ def assert_raises_regex(expected_exception, expected_regexp,
             raise AssertionError("Should have raised %r" %
                                  expected_exception(expected_regexp))
 
-try:
-    SkipTest = unittest.case.SkipTest
-except AttributeError:
-    # Python <= 2.6, we still need nose here
-    SkipTest = nose.SkipTest
+SkipTest = pytest.skip
```

Comment:
I don't think this is an accurate replacement, look at pytest.skip source code, it raises an exception. It seems like a more correct replacement is `_pytest.runner.Skipped`.

The Python 2.6 removal is going to happen soon, so maybe do something similar in the mean time:

```py
try:
    SkipTest = unittest.case.SkipTest
except AttributeError:
    SkipTest = _pytest.runner.Skipped
```